### ISSUE TITLE
"node14 is not supported"

### ISSUE THREAD

**Issue by johnpryan** [cedx/setup-dart#5]
I'm seeing this error now: 

```
Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node14' is not supported, use 'docker' or 'node12' instead.')
```

https://github.com/johnpryan/trie_router/runs/1443705749

Here's the configuration I'm using:

```yaml
name: Dart CI

on:
  push:
    branches: main
  pull_request:

defaults:
  run:
    shell: bash

jobs:
  test:
    name: "Dart Tests"
    runs-on: ubuntu-latest
    steps:
      - uses: cedx/setup-dart@v2
        with:
          release-channel: dev
      - uses: actions/checkout@v2
      - run: dart pub get
      - run: dart format --output=none --set-exit-if-changed .
      - run: dart analyze
      - run: dart test -r expanded

```

**Comment by johnpryan** [comment]
It looks like this is fixed by https://github.com/cedx/setup-dart/commit/0c31b5c99ec3d322715d320e4a644da9a51408d4

**Comment by cedx** [comment]
Thanks for reporting this issue. As you noted, it is fixed in version 2.4.1.
